src/OrbitGl/App.cpp

```diff
@@ -1808,7 +1808,7 @@ orbit_base::Future<void> OrbitApp::LoadSymbolsManually(
 
     // Explicitely do not handle the result.
     Future<void> future = RetrieveModuleAndLoadSymbolsAndHandleError(module).Then(
-        &immediate_executor, [](const SymbolLoadingAndErrorHandlingResult & /*result*/) -> void {});
+        &immediate_executor, [](const SymbolLoadingAndErrorHandlingResult& /*result*/) -> void {});
```

Comment:
This will trigger the clang-format check to fail. This is because the CI uses a different clang-format version as our machines. You need to change this by hand

Reply:
Thanks. Fixed.